build.sbt

```diff
@@ -14,6 +14,7 @@ lazy val embeddedKafka = "io.github.embeddedkafka" %% "embedded-kafka" % embedde
 
 inThisBuild(
   List(
+    version                  := "v0.17.3-SNAPSHOT",
```

Comment:
- no ` v` prefix
- `-cdk-SNAPSHOT` suffix

Reply:
Ok when I read cdk, I have that in mind: https://www.cdk.fr/ (from a long time ago where I would do unycicle lol).